tests/e2e-playwright/specs/admin/teacher/course-creation.spec.ts

```diff
@@ -10,7 +10,7 @@ import { teacherRole } from '@e2e/helpers/context';
 
 const { describe, use } = test;
 
-describe.parallel( 'Create Courses', () => {
+describe.skip( 'Create Courses', () => {
```

Comment:
👌 
I hope we'll fix it soon, as it is, probably, one of the most critical tests :)